lib/Net/Stripe/Constants.pm

```diff
@@ -0,0 +1,11 @@
+package Net::Stripe::Constants;
+
+use strict;
+
+use constant UNSUPPORTED_API_VERSION_PRE  => '2012-09-24'; # valid API version, but less than min supported
+use constant MIN_API_VERSION              => '2012-10-26';
+use constant INVALID_API_VERSION          => '2012-10-27'; # valid date within range, but invalid API version
```

Comment:
good catch. will fix and revert.

Reply:
any preference as to `use constant` vs `my` for these values within the test script?

also, a note as to the original reasoning for placement in `Net::Stripe::Constants`: even though not strictly necessary, the process of maintaining/updating these values, since they are all relative to one another, is eased by having them all in one place for easy visual comparison.
ie, as you update `MIN_API_VERSION`, you change `UNSUPPORTED_API_VERSION_PRE` to the previous value of `MAX_API_VERSION`, and you update `INVALID_API_VERSION` to the next calendar date after `MIN_API_VERSION`...